### PR TITLE
Contracts as npm dependency

### PR DESCRIPTION
**Pull request by muzam1l**

### CODE REVIEW

**Review by JaniAnttonen — COMMENTED** [review]
Let's wait for the contracts repo to be on npm and then publish this with it as the dependency :)

File: src/contracts.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import { FeeAmount } from '@uniswap/v3-sdk'
 import { Contract, providers, Signer } from 'ethers'
-import { ERC20 } from 'vanilla-contracts/typechain/vanilla_v1.1/ERC20'
-import { ERC20__factory } from 'vanilla-contracts/typechain/vanilla_v1.1/factories/ERC20__factory'
-import { VanillaV1Router02__factory } from 'vanilla-contracts/typechain/vanilla_v1.1/factories/VanillaV1Router02__factory'
-import { VanillaV1Router02 } from 'vanilla-contracts/typechain/vanilla_v1.1/VanillaV1Router02'
+import { MockERC20 as ERC20 } from '@vanilladefi/trade-contracts/typechain/vanilla_v1.1/MockERC20'
```
Comment (muzam1l) [inline src/contracts.ts]: Not sure about this, but is `MockERC20` replacement for `ERC20` for now and supposed to be used like this @JaniAnttonen?

Comment (JaniAnttonen) [reply src/contracts.ts]: Nnope, regular ERC20 contract should be found in there still :)

Comment (muzam1l) [reply src/contracts.ts]: Ok reverted, but ERC20 and factory are not present in npm package. Only present in local setup of contracts repo 🤔.

Comment (muzam1l) [reply src/contracts.ts]: Got it, typechains are not updated on releases in contracts repo because `postinstall` script does not run on npm package step!

**Comment by muzam1l** [comment]
Checks should pass now once latest version of contracts is published on npm.

**Pull request merged**